playbook/app/pb_kits/playbook/pb_line_graph/line_graph.rb

```diff
@@ -27,11 +27,16 @@ def chart_type
         gradient ? "area" : "line"
       end
 
+      def dark_theme
+        dark ? "dark" : ""
+      end
+
```

Comment:
I don't think this logic warrants it's own method

```suggestion

```

Reply:
So you think we shouldn't abstract the logic and pass a ternary directly?

Reply:
Also, I'm curious why is that better than writing the method?